src/git-promote.rs

```diff
@@ -0,0 +1,4 @@
+
+fn main() {
+    println!("promotet p");
```

Comment:
I'd recommend leaving this file (and Cargo.toml entry) out if the implementation is absent.  The other binary is a good enough example to go by, so the scaffolding value seems outweighed by the "mild false advertising" effect.